docs/sources/installation/docker.md

```diff
@@ -268,29 +192,36 @@ services:
     user: "104"
 ```
 
-#### Modifying permissions
+#### Modify permissions
 
-The commands below will run bash inside the Grafana container with your volume mapped in. This makes it possible to modify the file ownership to match the new container. Always be careful when modifying permissions.
+The commands below runs bash inside the Grafana container with your volume mapped in. This makes it possible to modify the file ownership to match the new container. Always be careful when modifying permissions.
 
 ```bash
 $ docker run -ti --user root --volume "<your volume mapping here>" --entrypoint bash grafana/grafana:5.1.0
 
 # in the container you just started:
 chown -R root:root /etc/grafana && \
-  chmod -R a+r /etc/grafana && \
-  chown -R grafana:grafana /var/lib/grafana && \
-  chown -R grafana:grafana /usr/share/grafana
+chmod -R a+r /etc/grafana && \
+chown -R grafana:grafana /var/lib/grafana && \
+chown -R grafana:grafana /usr/share/grafana
 ```
 
-## Migration from a previous version of the docker container to 6.4 or later
+### Migration from a previous version of the Docker container to 6.4 or later
 
-Grafana’s docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
+Grafana Docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
 
-## Migration from a previous version of the docker container to 6.5 or later
+### Migration from a previous version of the docker container to 6.5 or later
```

Comment:
I think we should change the ordering here to reverse chronological for the migration instructions. The latest migration instruction is most likely to be relevant to the most users.

Reply:
I'm game. It just means that we need to apply this change everywhere else to be consistent.